create_database.sql

```diff
@@ -1,8 +1,8 @@
 CREATE USER 'usermanagement'@'%' IDENTIFIED BY 'YOUR_SUPER_SECRET_PASSWORD';
 CREATE USER 'inventorymanagement'@'%' IDENTIFIED BY 'YOUR_SUPER_SECRET_PASSWORD';
 
-DROP DATABASE IF EXISTS inventorymanagement;
-DROP DATABASE IF EXISTS usermanagement;
```

Comment:
gehört das file nicht weg und auch aus dem dockerfile raus?

Reply:
Siehe Kommentar https://github.com/InventoryManagement2023/InventoryManagement-Backend/pull/2#discussion_r1161250157
Datei wird noch benötigt und auch in Docker Compose verwendet. Ich hab jetzt zusätzlich noch die Beispiel Insert Statements aus der Datei entfernt (macht keinen Sinn dort Daten einzufügen wenn die Tabellen noch nicht einmal existieren) und in eine separate Datei example_data.sql verschoben